src/sql/secrets.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE secrets (
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder if we should use a brand-new database for this. Or perhaps re-use the existing `account`. What do you think @j-f1 ?

Reply:
Hm… I think since there are already a bunch of things in the account db, I think we should add this table there instead of as a separate DB.